package.json

```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
     "@actions/artifact": "^0.2.0",
     "@actions/glob": "^0.1.0",
+    "@types/node": "^14.14.31",
```

Comment:
Where did this come from?

Reply:
Its not a problem, that its added, I just want to be sure I understand why it was added, before merging.

Reply:
For any folks just passing by, @andrewsanchez & I connected offline - sounds like he wrote his responses as a PR review, and didn't submit until after my second post. Interestingly enough, GH still maintains the insertion order of the comments, even though you'll see that the timestamps don't match up at all. TIL!